hooks/useMarketClosed.ts

```diff
@@ -17,9 +17,9 @@ const useMarketClosed = (currencyKey: CurrencyKey | null) => {
 			: false;
 
 	return {
-		isMarketClosed: isCurrencySuspended,
+		isMarketClosed: true, // isCurrencySuspended,
 		isCurrencySuspended,
-		marketClosureReason: currencySuspendedQuery.data?.reason as MarketClosureReason,
+		marketClosureReason: 'market-closure' as MarketClosureReason, // currencySuspendedQuery.data?.reason as MarketClosureReason,
```

Comment:
Assuming this is what we need to remove for hard-coding?

Reply:
yeah thats right, also in the `useGetFuturesMarkets` - its this commit that details the forced markets:

https://github.com/Kwenta/kwenta/pull/692/commits/3aeafbe347bad037849d24a62808efa7d0d2ed9f